webpack.config.js

```diff
@@ -1,19 +1,21 @@
+const webpack = require('webpack');
+
 module.exports = {
-    entry: "./browser.js",
+    entry: './browser.js',
     output: {
         path: __dirname,
-        filename: "browser.bundle.js"
+        filename: "build/browser.bundle.js"
     },
     module: {
         loaders: [
             {
                 test: /\.js$/,
                 exclude: /(node_modules|bower_components)/,
-                loader: 'babel'//, // 'babel-loader' is also a legal name to reference
-                //query: {
-                    //presets: ['es2015']
-                //}
+                loader: 'babel'
             }
         ]
-    }
+    },
+    plugins: [
+        new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /en|ru/)
```

Comment:
now moment is built with en and ru locales only, so we can just run `npm run build` before commit without changing `Tranform.js` 🐗